2019-10-23 10:11:08,903 ERROR: Exception on /user/Matias [GET] [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/routes.py", line 84, in user
    next_url=next_url, prev_url=prev_url)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/templates/user.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/templates/base.html", line 31, in top-level template code
    {% block content %}{% endblock %}
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/templates/user.html", line 27, in block "content"
    {% for post in posts %}
TypeError: 'Pagination' object is not iterable
2019-10-23 10:12:02,998 ERROR: Exception on /user/Matias [GET] [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/routes.py", line 84, in user
    next_url=next_url, prev_url=prev_url)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/templates/user.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/templates/base.html", line 31, in top-level template code
    {% block content %}{% endblock %}
  File "/Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/templates/user.html", line 27, in block "content"
    {% for post in posts %}
TypeError: 'Pagination' object is not iterable
2019-10-23 10:15:29,309 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:44]
2019-10-23 10:48:49,249 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:47]
2019-10-23 10:53:17,703 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:47]
2019-10-23 11:25:08,683 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:47]
2019-10-23 11:48:01,404 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:47]
2019-10-23 11:48:24,616 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:47]
2019-10-23 11:57:03,858 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:47]
2019-10-23 12:07:28,363 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:47]
2019-10-23 18:51:20,622 INFO: Microblog startup [in /Users/usr_python/GDcocamatias/Desarrollo de SW/Cursos/Flask/microblog/app/__init__.py:47]
